.github/workflows/release.yml

```diff
@@ -155,7 +136,11 @@ jobs:
           EOL
 
       - name: "Create draft release"
-        run: echo "RELEASE_URL=$(gh release create ${{ inputs.version }} --target ${{ github.ref_name }} --title "${{ inputs.version }}" --notes-file release-message --draft)" >> "$GITHUB_ENV"
+        run: |
+          if [[ "${{ inputs.version }}" =~ (alpha|beta) ]]; then
```

Comment:
> Note that since PECL doesn't have a stability constant for release candidates, so I haven't yet added support for them. However, it should be easy enough to add in case we want to do so.

I'll take this as a "yes, let's support release candidates". I'll update the tooling accordingly 👍